examples/ode_models/ODE_Lotka_Volterra_multiple_ways.myst.md

```diff
@@ -0,0 +1,963 @@
+---
+jupytext:
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
+---
+
++++ {"tags": []}
+
+(ODE_Lotka_Volterra_fit_multiple_ways)= 
+# ODE Lotka-Volterra With Bayesian Inference in Multiple Ways
+
+:::{post} January 10, 2023
+:tags: ODE, pytensor op, gradient-free inference, pytensor scan
+:category: intermediate, how-to
+:author: [Greg Brunkhorst]{https://github.com/gbrunkhorst}
```

Comment:
This is only rendered as plain text at the very bottom of the file for the citation template, so it should be your name in plain text. You should render your name as link to your github profile at the text of the "Authors" section.

Reply:
revised